### PR TITLE
Update RADIUS default mapping

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

File: src/ergw_aaa_radius.erl [hunk 1]
```diff
@@ -37,12 +37,12 @@
...
-    {upf_failure, 2},
-    {session_error, 2},
+    {administrative, 10},
+    {link_broken, 9},
+    {upf_failure, 9},
+    {session_error, 9},
     {inactivity_timeout, 4},
-    {path_restart, 2}
+    {path_restart, 9}
 ]).
```
Comment (RoadRunnr) [inline src/ergw_aaa_radius.erl:46]: I'm not convinced that this correct, not matter what a customer might want, the default should be reasonably allowed with RFC 2866.

e.g. administrative should really be 6, link_broken could be 2 or 3.

**Review by RoadRunnr — CHANGES_REQUESTED** [review]
using this mapping as default is IMHO wrong.

**Comment by vkatsuba** [comment]
> using this mapping as default is IMHO wrong.

Agree. I was thinking maybe make sense create default mapping with names of reasons and with values based on RFCs and create some logic for configuration mapping for places where we use those reasons in ergw, eg we use in context_idle reason `inactivity_timeout` with value `4` and in future when we will need to change `inactivity_timeout` name eg to `idle_timeout` with the same or with the other value - we will need to change code in few places... So, maybe make sense create more flexible logic for configuration like `[#{place => context_idle,  reason_name => idle_timeout, reason_value => 2}]`(also in ergw we have a few places where we use the same termination causes name and in future maybe we will need change eg `administrative` to other termination cause in delete_context but keep `administrative` in other place), but by default use mapping from RFC... Also, if my point make sense for you, I not sure if we need implement it in current implementation, this should be something like feature change request or something like it.

File: src/ergw_aaa_radius.erl (hunk 1 above)
Comment (vkatsuba) [reply src/ergw_aaa_radius.erl:46]: Done

File: src/ergw_aaa_radius.erl [hunk 2]
```diff
@@ -39,10 +39,10 @@
     {normal, 1},
     {administrative, 6},
     {link_broken, 2},
-    {upf_failure, 2},
-    {session_error, 2},
+    {upf_failure, 9},
+    {remote_failure, 9},
```
Comment (RoadRunnr) [inline src/ergw_aaa_radius.erl:43]: I'm still not convinced that code 9 `NAS Error` is correct. IMHO 2 `Lost Carrier` is more appropriate.

@mgumz what is you take on this? Do we have a document describing our default mapping (I don't care what customers might thing, we can configure a non standard mapping for them).

**Review by fholzhauser — APPROVED** [review]

**Pull request merged**